apps/web/src/pages/settings/organization/OrganizationLogo.tsx

```diff
@@ -0,0 +1,65 @@
+import { errorMessage } from '@novu/design-system';
+import { MimeTypesEnum } from '@novu/shared';
+import React, { useEffect } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { ProfileImage } from '../../../components/shared';
+
+type FormValues = {
+  logoUrl: string;
+};
+
+const MIME_TYPES = {
+  'image/jpeg': MimeTypesEnum.JPEG,
+  'image/png': MimeTypesEnum.PNG,
+};
+
+export function OrganizationLogo({ logoUrl }: { logoUrl?: string }) {
```

Comment:
```suggestion
export function OrganizationLogo({ logoUrl }: Partial<FormValues>) {
```
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** Could do this if we want to correlate the form with optional props‏